sympy/liealgebras/weyl_group.py

```diff
@@ -9,7 +9,7 @@ class WeylGroup(Basic):
 
     """
     For each semisimple Lie group, we have a Weyl group.  It is a subgroup of
-    the isometry group of the root system.  Specifically, it’s the subgroup
+    the isometry group of the root system.  Specifically, it's the subgroup
```

Comment:
This file still has the coding line at the top.

Reply:
Oh I guess there are Unicode printing characters (in the second to last line of the file).

Reply:
Yeah this file and a couple of other in liealgebras are still whitelisted because of this:
https://github.com/sympy/sympy/blob/3fe9b14c1f8839bca921b656b004518408db93c8/sympy/liealgebras/weyl_group.py#L401-L403